gateway/ln-gateway/src/lib.rs

```diff
@@ -93,28 +90,13 @@ impl Gateway {
         // Create message channels for the webserver
         let (sender, receiver) = mpsc::channel::<GatewayRequest>(100);
 
-        // Source route hints form the LN node
-        let mut num_retries = 0;
-        let route_hints = loop {
-            let route_hints: Vec<RouteHint> = lnrpc
-                .routehints()
-                .await
-                .map_err(|_| GatewayError::FailedToFetchRouteHints)?
-                .try_into()
-                .map_err(|_| GatewayError::FailedToFetchRouteHints)?;
-
-            if !route_hints.is_empty() || num_retries == ROUTE_HINT_RETRIES {
-                break route_hints;
-            }
-
-            info!(
-                ?num_retries,
-                "LN node returned no route hints, trying again in {}s",
-                ROUTE_HINT_RETRY_SLEEP.as_secs()
-            );
-            num_retries += 1;
-            tokio::time::sleep(ROUTE_HINT_RETRY_SLEEP).await;
-        };
+        // Fetch route hints form the LN node
+        let route_hints: Vec<RouteHint> = lnrpc
```

Comment:
lgtm. I'll let @elsirion comment on the value of the retry. Seems like this only applied to some aspect of CLN plugin